test.js

```diff
@@ -228,7 +228,7 @@ describe('prettyFormat()', function() {
 
   it('should support plugins', function() {
     'use strict';
-    class Foo {};
+    function Foo() {};
```

Comment:
Actually I change this, but I add a couple of arrow function, so it will still fail I guess... 
unless ci runs with --harmony and it was already in 0.12